hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -343,22 +595,209 @@
          :read (assoc op :type :ok, :value (into (sorted-set) (.get m "hi")))))
 
      (teardown! [this test]
-       (.terminate conn)))))
+       (.shutdown conn)))))
 
 (defn map-workload
   "A workload for map tests, with the given client options."
   [client-opts]
-  {:client    (map-client client-opts)
-   :generator (->> (range)
-                   (map (fn [x] {:type  :invoke
-                                 :f     :add
-                                 :value x}))
-                   gen/seq
-                   (gen/stagger 1/10))
+  {:client          (map-client client-opts)
+   :generator       (->> (range)
+                         (map (fn [x] {:type  :invoke
+                                       :f     :add
+                                       :value x}))
+                         gen/seq
+                         (gen/stagger 1/10))
    :final-generator (->> {:type :invoke, :f :read}
                          gen/once
                          gen/each)
-   :checker   (checker/set)})
+   :checker         (checker/set)})
+
+
+
+(defn parseLine [line]
```

Comment:
Functions in Clojure are generally kebab-case. Not a huge deal, this is perfectly legal, just letting you know in case you're looking for style feedback. You may also want to add some docstrings here, because it's not totally clear what format this expects, or what it's intended to do.